changelog.d/pa-2263.fixed

```diff
@@ -0,0 +1,2 @@
+DeepSemgrep: Semgrep no longer crashes in certain cases when `metavariable-pattern/regex`
```

Comment:
Changelog entry needs update? Not DeepSemgrep specific. Although this is less important, there was never any crash. This actually affected autofix as well. So it's generally about incorrect behavior when metavariables matched imported names.